conda/recipes/cugraph-pyg/meta.yaml

```diff
@@ -35,6 +35,9 @@ requirements:
     - numba>=0.56.2
     - numpy
     - pytorch
+    # TODO: Determine if `cudatoolkit` is a necessary runtime dependency for this package.
+    # If so, the version spec may need to be changed to `11.0>=,<12.0` to match the rest of our packages.
+    # If not, we should remove `cudatoolkit` and update the version string above accordingly.
```

Comment:
cc: @alexbarghi-nv, do you have thoughts on this?

Reply:
I believe it is a necessary runtime dependency, and should be >= 11.5

Reply:
@ajschmidt8 based on @alexbarghi-nv's comment, this TODO should be removed and `cudatoolkit` should have a pin for `>11.5`

Reply:
removed this in 3ff9bfa. thanks @alexbarghi-nv and @rlratzel.